ARC specification.md

```diff
@@ -493,14 +493,15 @@ As the ARC might be used by different persons and in different workflow contexts
 An ARC can be augmented into an RO Crate by placing a metadata file `ro-crate-metadata.json` into the top-level ARC folder, which must conform to the [RO Crate specification](https://www.researchobject.org/ro-crate/1.1/).
 The ARC root folder is then simultaneously the RO Crate Root and represents an ISA investigation.
```

Comment:
Ofc not changed in this PR, but.. 

Is this technically even possible? Or do we have to put the Investigation also as a property into the RO-Crate Root?

Reply:
The root data entity should be the root directory, which corresponds to the ISA investigation. Thus, I would keep it as is.